### PR TITLE
WCAG - Changes to attributes and update `dialog` spelling

### PR DESCRIPTION
**Pull request by chadwickmeyer**

### CODE REVIEW

File: packages/angularjs-extras/src/controllers/panel.js [hunk 1]
```diff
@@ -34,8 +34,8 @@
 
       // Digest Model Bindings
       $scope.model = null
-      if ($element.attr('ng-model')) {
-        $scope.$parent.$watch($element.attr('ng-model'), function (model) {
+      if ($element.attr('data-ng-model')) {
```
Comment (alexgurrola) [inline packages/angularjs-extras/src/controllers/panel.js:37]: This should check for both, as technically both are valid possibilities.

File: packages/angularjs-extras/src/controllers/panel.js [hunk 2]
```diff
@@ -34,8 +34,8 @@
...
+        $scope.$parent.$watch($element.attr('data-ng-model'), function (model) {
```
Comment (alexgurrola) [inline packages/angularjs-extras/src/controllers/panel.js:38]: This should check for both, as technically both are valid possibilities.

File: packages/angularjs-extras/src/controllers/panel.js [hunk 3]
```diff
@@ -44,8 +44,8 @@
 
       // Digest Collection Bindings
       $scope.collection = null
-      if ($element.attr('ng-collection')) {
-        $scope.$parent.$watch($element.attr('ng-collection'),
+      if ($element.attr('data-ng-collection')) {
```
Comment (alexgurrola) [inline packages/angularjs-extras/src/controllers/panel.js:47]: This should check for both, as technically both are valid possibilities.

File: packages/angularjs-extras/src/controllers/panel.js [hunk 4]
```diff
@@ -44,8 +44,8 @@
...
+        $scope.$parent.$watch($element.attr('data-ng-collection'),
```
Comment (alexgurrola) [inline packages/angularjs-extras/src/controllers/panel.js:48]: This should check for both, as technically both are valid possibilities.

**Review by alexgurrola — APPROVED** [review]

**Review by alexgurrola — APPROVED** [review]

File: packages/angularjs-extras/src/controllers/panel.js (hunk 2 above)
Comment (chadwickmeyer) [reply packages/angularjs-extras/src/controllers/panel.js:38]: @alexgurrola In an effort to simplify and standardize (we don't want to support multiple variations) I thought it was better to use only the valid version, only `data-*` is valid by the standards. So it makes no sense to allow a non-standard `ng-x` as well. If someone tries to use ng-x and it doesn't work, it should be obvious that they didn't use a valid attribute. Right?

File: packages/angularjs-extras/src/controllers/panel.js (hunk 3 above)
Comment (chadwickmeyer) [reply packages/angularjs-extras/src/controllers/panel.js:47]: See other comment. I don't believe both are valid, only data-x is a valid HTML attribute.

File: packages/angularjs-extras/src/controllers/panel.js (hunk 1 above)
Comment (chadwickmeyer) [reply packages/angularjs-extras/src/controllers/panel.js:37]: See other comments about this.

File: packages/calendar/src/eventDialog.html [hunk 5]
```diff
@@ -1,5 +1,5 @@
 <md-dialog aria-label="{{::ctrl.eventData.title + ' Event Dialog'}}">
-    <article id="calendar-event-dialogue" class="initialize-dialogue dialogue-container background-light default-border-radius default-shadow">
+    <article id="calendar-event-dialog" class="initialize-dialog dialog-container background-light default-border-radius default-shadow">
```
Comment (apocist) [inline packages/calendar/src/eventDialog.html:2]: this change will affect any css stylings points to this id and classes

Comment (chadwickmeyer) [reply packages/calendar/src/eventDialog.html:2]: Yes, but I did a search and there was no other references with classnames for the old 'dialogue'. So initialize-dialogue and initialize-dialogue-container are probably not currently used.

**Pull request merged**